### PR TITLE
[SVN] removed %b identifier to fix color encoding

### PR DESCRIPTION
**Pull request by shibumi**
This is a workaround for https://github.com/bhilburn/powerlevel9k/issues/312 I've just added custom svn prompts instead of using our primary ones. The problem is the `%b`-identifier. When we use the `%b`-identifier for svn it seems to break our color-encoding. I guess they use some chars in there or some custom color encoding that will  break ours. The `%b`-identifier will give for SVN the following output:
`<branch-name>:<revision number of branch>`

When I use `%b` everything after `<branch-name>` will be yellow..I tried to fix the color with it. But maybe someone else has more luck.

It's sad that we lose the branch-name for this feature. But I think most people use only one branch with svn and we still display the revision-number. So it shouldn't be that bad. Let's focus on `git`. :)

The only thing what I would still miss for SVN is a nicer icon. I thought about using the `VCS_BRANCH_ICON` for this or we wait for `nerdfonts`-support.. I guess they have some cool icon for svn.

tl;dr: The svn colors are fixed now, I just removed the`%b`-identifier from the svn-bar.

### CODE REVIEW

**Comment by bhilburn** [comment]
Nice work, @shibumi! #312 has been around for a while, now, and when I looked at it my head imploded.

Thanks for the great contribution =)
